code/game/objects/structures/ghost_role_spawners.dm

```diff
@@ -43,7 +43,7 @@
 	outfit = /datum/outfit/ashwalker
 	roundstart = FALSE
 	death = FALSE
-	anchored = FALSE
+	anchored = TRUE
```

Comment:
👀 

will eggs still fall down chasms created by the destruction of the tendril?

Reply:
It won't, the ashwalker tendril is special as it only spawns a chasm directly on it's own tile.

Reply:
This was changed because someone expressed concern about Ash Walkers just dragging eggs with them to the station for an easier respawn, which I feel isn't really a healthy mechanic.

Regardless, mechs are anchored and fall into a chasm, so eggs that are above one will fall too.

Reply:
If you do this you should allow people to destroy the eggs somehow. Currently the way to do it is to push them into the chasm.

Reply:
This has been reverted.